src/main/kotlin/com/kevinlinxp/sublimeSnippetsSupport/SublimeSnippetScope.kt

```diff
@@ -30,6 +30,7 @@ enum class SublimeSnippetScope(private val scopeName: String, private val contex
     SOURCE_TSX("source.tsx", "TypeScript"),
     SOURCE_LUA("source.lua", "LUA"),
     SOURCE_PUPPET("source.puppet", "PUPPET_FILE"),
+    SOURCE_OTHER("source.other", "OTHER"),
```

Comment:
The scope of "OTHER" is ambiguous per IDE and per individual. So it's better not to add this scope which may bring in unwanted snippets.

Reply:
You're right  right! I removed it from this request